security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -223,7 +230,9 @@ void run(CommandLine line, ToolRunningContext context, ContingenciesProviderFact
 
         context.getOutputStream().println("Loading network '" + caseFile + "'");
         Properties inputParams = readProperties(line, ConversionToolUtils.OptionType.IMPORT, context);
-        Network network = Importers.loadNetwork(caseFile, context.getShortTimeExecutionComputationManager(), ImportConfig.load(), inputParams);
+        boolean skipPostProc = line.hasOption(SKIP_POSTPROC_OPTION);
+        ImportConfig importConfig = (!skipPostProc) ? ImportConfig.load() : new ImportConfig();
```

Comment:
I’m not sure this will always be equivalent. It’s equivalent right now because there is only a single property in Import module?
Is it possible to load the configuration, and to remove the post processor if the option is set?

You also need to update the documentation of this tool

Reply:
I copied the skipPostProc mecanism already present in RunLoadFlowTool and SensitivityComputationTool... If this mecanism should be changed, maybe we should do it in another PR?